app-text/zathura/zathura-0.5.11.ebuild

```
@@ -1,44 +1,43 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit meson virtualx xdg
 
-DESCRIPTION="A highly customizable and functional document viewer"
+DESCRIPTION="Highly customizable & functional document viewer"
 HOMEPAGE="https://pwmt.org/projects/zathura/"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
 	EGIT_BRANCH="develop"
```

```suggestion
	EGIT_REPO_URI="https://github.com/pwmt/zathura.git"
```

See : Homepage and https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=ec6b828c775289bfead2437acf7e816356bc93ae